docker-compose.testing.yml

```diff
@@ -4,6 +4,7 @@ services:
   nuxt:
     environment:
       API_HOST: http://nuxt:3000
+      LIKECOIN_AUTH_URL:
```

Comment:
Missing ?

Reply:
Need to wait for user login integration test.